tests/test_charge_point.py

```diff
@@ -268,6 +268,8 @@ async def test_services(hass, socket_enabled):
         # use same id to ensure metrics populated
         cp = ChargePoint("CP_1_test", ws)
         cp.accept = False
+        cs.charge_points[cs.cpid].received_boot_notification = False
+        cs.charge_points[cs.cpid].post_connect_success = False
```

Comment:
@lbbrhzn, not sure why these statements do not fully test the if statements above, any ideas?

Reply:
looks like the NotImplementedError is not thrown.

Reply:
I'm not sure how the  NotImplementedError should work, there is also the exceptions.py code which is not called during testing but references throwing an error configuring the charger.